lib/ferrum/browser/options/chrome.rb

```diff
@@ -63,7 +63,23 @@ def merge_required(flags, options, user_data_dir)
                       "user-data-dir" => user_data_dir)
         end
 
+        def ensure_required!(options)
+          return if options[:browser_options].nil?
+
+          %w[remote-debugging-port remote-debugging-address
+             window-size user-data-dir].each do |required|
+            next unless options[:browser_options].key?(required)
+
+            if options[:browser_options][required] == false ||
+               options[:browser_options][required].nil?
+              raise ArgumentError, "#{required} is required"
+            end
+          end
+        end
```

Comment:
This method is a good candidate to be in `lib/ferrum/browser/options/base.rb` to follow the DRY principle.

Reply:
@Mifrill Agree. I made the change and updated the PR to avoid this repetition. Thanks for pointing it out!